src/components/Dashboard/Form/Button/Button.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import Button from "./Button";
+
+describe("Button", () => {
```

Comment:
Button component can use a lot of props what create a many tests cases. Please add more tests e.g. if a component change label basing on props `label`

Reply:
I added, please check